packages/core-cordova/www/metrics.js

```diff
@@ -0,0 +1,57 @@
+const MetricsService = cordova.require("core.core").MetricsService;
+
+const MOBILE_CORE_CLASS = "MobileCoreModule";
+
+class MetricsImpl {
+
+  constructor(id, data) {
+    this.id = id;
+    this.data = data;
+  }
+
+  collect() {
+    return this.data;
+  }
+}
+
+class CordovaMetricsService extends MetricsService {
+
+  getAppAndDeviceMetrics() {
+    return new Promise((res, rej) => {
+      cordova.exec(
+        metrics => {
+          const plugins = cordova.require("cordova/plugin_list");
+          const sdkVersion = plugins.metadata["core-cordova"];
+          metrics.app.sdkVersion = sdkVersion;
```

Comment:
SDK version should be done inside the app metrics.

Reply:
app metrics is just a plain JSON returned by the Android module, are you suggesting we add a wrapper class around it? IMO we could leave it like this for now, to keep things a little simpler.

Reply:
Yeah. I have noticed that we wrap that payload bellow into interfaces.